packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1669,7 +1669,10 @@ async function handler(
           metadata.conferenceData = updatedEvent.conferenceData;
           metadata.entryPoints = updatedEvent.entryPoints;
           handleAppsStatus(results, booking);
-          videoCallUrl = metadata.hangoutLink || videoCallUrl;
+          // @ts-expect-error - safe to ignore the below error as we are parsing the data with zod
+          const parseUpdatedEventUrl = z.string().safeParse(updatedEvent?.url);
```

Comment:
I don't think we need to use zod here as it's just a string. Instead, we need to fix the type for `updatedEvent` to include url in it.